oauthdoctor/bin/gen-bin.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
There are a number of changes. We need to adhere to 
https://www.shellcheck.net/

Copy your code and lint it their for the details.